internal/weldr/store.go

```diff
@@ -148,6 +149,12 @@ func (s *store) getBlueprint(name string, bp *blueprint, changed *bool) bool {
 	if bp.Modules == nil {
 		bp.Modules = []blueprintPackage{}
 	}
+	if bp.Groups == nil {
+		bp.Groups = []blueprintPackage{}
+	}
+	if bp.Version == "" {
+		bp.Version = "0"
```

Comment:
I've seen lorax-composer complain about version that don't follow "semantic versioning". Does cockpit-composer depend on that as well? if so, it might make sense to put `0.0.0` here and below.

Reply:
Cockpit-composer does not depend on semantic versioning. It just requires the version to be a string. I can change it to semantic versioning to be in line with lorax composer.

Reply:
The version is semantic now